packages/gatsby-transformer-cloudinary/gatsby-node.js

```diff
@@ -10,12 +10,38 @@ const {
 } = require('./gatsby-image');
 
 //import plugin options, used to check for API key before uploading assets to Cloudinary
+
+let coreSupportsOnPluginInit = undefined;
+
+try {
+  const { isGatsbyNodeLifecycleSupported } = require(`gatsby-plugin-utils`);
+  if (isGatsbyNodeLifecycleSupported(`onPluginInit`)) {
+    coreSupportsOnPluginInit = 'stable';
+  } else if (isGatsbyNodeLifecycleSupported(`unstable_onPluginInit`)) {
+    coreSupportsOnPluginInit = 'unstable';
+  }
+} catch (error) {
+  console.error(
+    `Cannot check if Gatsby supports onPluginInit lifecycle: 💜 🐸 on🔌👸 life🚴‍♀️ `
+  );
+}
+
 const pluginOptions = getPluginOptions();
 
-exports.onPreInit = ({ reporter }, pluginOptions) => {
+const initializaGlobalState = (_, pluginOptions, { reporter }) => {
```

Comment:
The params need to be the same as for onPreInit, ie. first deconstruct reporter from the gatsbyUtils, then catch the plugin options.

```
const initializaGlobalState = ({ reporter }, pluginOptions) => {
```

Reply:
What is the `_,` param?